src/collection-preferences/index.tsx

```diff
@@ -77,7 +79,7 @@ export default function CollectionPreferences({
     contentDensityPreference ||
     customPreference
   );
-  const hasRightContent = !!visibleContentPreference;
+  const hasRightContent = !!(visibleContentPreference || contentDisplayPreference);
```

Comment:
Minor: I'd rename the var to `hasContentOnTheRight`

Reply:
Changed —also `hasLeftContent` as `hasContentOnTheLeft`.